src/libs/OptionsListUtils.js

```diff
@@ -352,6 +352,40 @@ function getNewChatOptions(
     });
 }
 
+/**
+ * Build the options for the New Chat view
+ *
+ * @param {Object} myPersonalDetail
+ * @param {String} amountText
+ * @returns {Array}
+ */
+function getDisplayOptionFromMyPersonalDetail(
```

Comment:
DisplayOptions is a bit vague. I think something like `getIOUConfirmationOptions` or `getIOUConfirmationOptionsFromPersonalDetails` would be slightly better. The first one is probably fine?

Reply:
I think second would be better because we use two display options for Confirmation. One is from Participants and one from MyPersonalDetail